ablySpec/RealtimeClient.connection.swift

```diff
@@ -713,6 +713,49 @@ class RealtimeClientConnection: QuickSpec {
                 }
 
             }
+
+            // RTN12
+            context("close") {
+
+                // RTN12c
+                it("close move to CLOSED state if transport is abruptly closed") {
```

Comment:
This description is not clear, it should be `transitions to the CLOSING state and then to the CLOSED state if the transport is abruptly closed`